app/Models/Role.php

```diff
@@ -60,6 +97,52 @@ public function name(): string
         return __('role.' . $this->name);
     }
 
+    /**
+     * Returns true if the role can be attached to only one user at a time.
+     * @param int $roleId
+     * @param string $objectName optional. Returns false if the object is null for a role which can have objects.
+     * @return bool
+     */
+    public static function isUnique($roleName, $objectName = null): bool
+    {
+        switch ($roleName) {
+            case self::DIRECTOR:
+                return true;
```

Comment:
nit, you could skip this return and the one in L112.

Reply:
Noo, i do not want that :D

Reply:
Why? That's the point of using a switch...

Reply:
I like the switches with implicit breaks